mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -206,11 +281,11 @@ impl ZRpcOps for LightRpcClient {
         true
     }
 
-    #[cfg(not(target_arch = "wasm32"))]
     async fn checkpoint_block_from_height(
         &mut self,
         height: u64,
     ) -> MmResult<Option<CheckPointBlockInfo>, UpdateBlocksCacheErr> {
+        info!("SYNC STARTING HEIGHT: {height}");
```

Comment:
Please provide more context in this log. Is it possible to also log coin ticker and other info?

Reply:
yes..done